.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -8,17 +14,25 @@ Are you looking for help with getting started on Reaction? Please visit our [Rea
 * [ ] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)
 
 ## Issue Description
+```
```

Comment:
This updates the issue template to have code blocks, not comment blocks.

See this gist for details on how to comment out bits in GitHub
https://gist.github.com/jonikarppinen/47dc8c1d7ab7e911f4c9

essentially, use 
`<!--` and `-->` to open and close a comment block.

```
<!-- 
Commented out instructions
--> 
```

Reply:
yeah, I see what you mean. Blocks updated now <!-- for real this time -->